source/ed/connectors/osm_tags_reader.cpp

```diff
@@ -224,4 +226,55 @@ std::bitset<8> parse_way_tags(const std::map<std::string, std::string> & tags){
     return result;
 }
 
-}} //namespace navitia::OSM
+PoiTypeParams::PoiTypeParams(const std::string& json_params) {
+    namespace pt = boost::property_tree;
+
+    pt::ptree poi_root;
+    std::istringstream iss (json_params);
+    pt::read_json(iss, poi_root);
+
+    for (const pt::ptree::value_type& json_poi_type: poi_root.get_child("poi_types")) {
+        std::string poi_type_id = json_poi_type.second.get<std::string>("id");
+        if (poi_types.find(poi_type_id) != poi_types.end()) {
+            throw std::invalid_argument("POI type id " + poi_type_id + " is defined multiple times");
+        }
+        poi_types[poi_type_id] = json_poi_type.second.get<std::string>("name");
```

Comment:
There is a try around the wall to that method to output the problem when just checking that json is OK. Not around the regular osm2ed call, because crashing seems OK to me in that case...
(if that was the question)